deployment/terraform/data.tf

```diff
@@ -0,0 +1,13 @@
+data "terraform_remote_state" "core" {
+  backend = "s3"
+
+  config {
+    region = "${var.aws_region}"
+    bucket = "${var.remote_state_bucket}"
+    key    = "terraform/core/state"
+  }
+}
+
+data "aws_route53_zone" "external" {
+  zone_id = "${data.terraform_remote_state.core.public_hosted_zone_id}"
```

Comment:
Either lookup will require a corresponding output in `geotrellis-site-deployment`, but I went with `zone_id` because that attribute is unique/less ambiguous to me.